README.md

```diff
@@ -16,7 +18,7 @@ For extended documentation see [here](https://developer.here.com/rest-apis/docum
 
 ### Installation
 
-Make sure you have already installed the latest version of Node and NPM (Node Package Manager) and then run:
+Make sure you have already installed the latest version of Node (at least version 4) and NPM (Node Package Manager) and then run:
```

Comment:
Please change to version 4.2.x due to 
```
npm WARN engine selenium-webdriver@2.53.2: wanted: {"node":">= 4.2.x"} (current: {"node":"3.0.0","npm":"2.13.3"})
```
when deliberately testing with a previous version